rundeckapp/grails-app/services/rundeck/services/ProjectService.groovy

```diff
@@ -1077,9 +1079,18 @@ class ProjectService implements InitializingBean, ExecutionFileProducer, EventPu
 
                 'files/' {
                     'etc/' {
-                        if (importConfig) {
+                        if (importConfig || importNodes) {
                             'project.properties' { path, name, inputs ->
                                 configtemp = copyToTemp()
+                                String resourceConfig = ""
+                                configtemp?.text?.eachLine {
+                                    if(it.contains("resources.source")){
```

Comment:
I think it would be better to exclude properties starting with `resources.source` after parsing it.  This check is not exact